update/ansible/playbook/tasks/roles/dashboards_upgrade/tasks/main.yml

```diff
@@ -83,10 +83,6 @@
       ignore_errors: true
       # FIXME: we have a race condition here. We generate grafana supervisor config in pmm-managed and it doesn't exist at this stage
 
-    - name: Remove old DBaaS dashboard (Before provisioning times)
-      command: sqlite3 /srv/grafana/grafana.db -cmd ".timeout 60000" "DELETE FROM dashboard WHERE title = 'DBaaS' AND slug = 'dbaas';"
-      changed_when: true
-
```

Comment:
we still need it  to delete old dashboards

Reply:
I'll revert it before the merge. No worries.

Reply:
Done